docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,1981 @@
+* Addss value count to *Lens* {kibana-pull}136385[#136385]
+* Adds standard deviation function in *Lens* {kibana-pull}136323[#136323]
+* Adds the ability to set the font size for mosaic outer level in *Lens* {kibana-pull}135911[#135911]
+* Adds the ability to rank top values by custom metric in *Lens* {kibana-pull}134811[#134811]
+* Adds the ability to convert TSVB series agg to *Lens* configuration {kibana-pull}134681[#134681]
+* Adds the ability to allow multiple split accessors {kibana-pull}134566[#134566]
+* Adds xExtent validation within expression {kibana-pull}134546[#134546]
+* Adds the ability to render newlines in data table in *Lens* {kibana-pull}134441[#134441]
+* Extends Axis bounds for XY chart when using Interval operation in *Lens* {kibana-pull}134020[#134020]
+* Adds the ability to use pick_max instead of clamp for positive only {kibana-pull}133460[#133460]
+* Adds a new pick_min/max operation and clamp fixes in *Lens* {kibana-pull}132449[#132449]
+* Adds support for percentile_ranks aggregation in *Lens* {kibana-pull}132430[#132430]
+* Implements the ability to drag and drop between layers in *Lens* {kibana-pull}132018[#132018]
+* Adds optimization for percentiles fetching in *Lens* {kibana-pull}131875[#131875]
+* Adds `axes` support {kibana-pull}129476[#129476]
+
+Machine Learning::
+* Adds tooltips for disabled actions in the Trained Models list {kibana-pull}137176[#137176]
+* Data visualizer: Add field types in-product help {kibana-pull}137121[#137121]
+* Plot zero scores on the Overall anomaly swim lane {kibana-pull}136951[#136951]
+* One-way cursor sync added from Anomaly detection swimlane to other charts {kibana-pull}136775[#136775]
+* Adds action to view datafeed counts chart to jobs list rows {kibana-pull}136274[#136274]
+* Data Visualizer: Remove duplicated geo examples, support 'version' type, add filters for boolean fields, and add sticky header to Discover {kibana-pull}136236[#136236]
+* Adds a link to ML trained models list from ID in Stack Management app table {kibana-pull}135700[#135700]
+* Adds information callouts to trained model testing flyout {kibana-pull}135566[#135566]
+* Limit Use full data button in anomaly detection job wizards to past data only {kibana-pull}135449[#135449]
+* Replace a fixed Y-axis width with a max width setting for Anomaly Swim Lane Embeddable {kibana-pull}135436[#135436]
+* Adds support for setting threading params when starting a trained model deployment {kibana-pull}135134[#135134]
+* Management page refactor {kibana-pull}134893[#134893]
+* Disable the Single Metric Viewer button for not viewable jobs {kibana-pull}134048[#134048]
+* Anomaly Detection: allow snapshot to be reverted from the view datafeed flyout {kibana-pull}133842[#133842]
+
+Management::
+* Transforms: Adds per-transform setting for num_failure_retries to creation wizard and edit flyout and authorization info {kibana-pull}135486[#135486]
+* Adds sorting to Transform's Audit Messages {kibana-pull}135047[#135047]
```

Comment:
For consistency with the formatting we normally use for transforms PRs, I edited the title of this PR to `Transforms: Add sorting to audit messages tab`